package.json

```diff
@@ -35,12 +36,11 @@
     "ts:check": "tsc --project tsconfig.json --noEmit",
     "ts:emit": "lerna run ts:emit",
     "prepublishOnly": "yarn lint && yarn test && yarn build",
-    "release": "lerna publish --no-commit-hooks",
+    "release": "yarn changeset release --public",
     "release:site": "gh-pages --dist storybook/public"
   },
   "husky": {
     "hooks": {
-      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
```

Comment:
Could we remove the commitlint-related dependencies too?

Reply:
👍